massa-grpc/src/stream/new_operations.rs

```diff
@@ -0,0 +1,183 @@
+                                                coins: coins.to_raw()
+                                            }),
+                                            ..Default::default()
+                                        }
+                                    },
+                                };
+
+                                let ret = grpc::SecureShareOperation {
+                                    content: Some(operation.content.into()),
+                                    //HACK do not map serialized_data
+                                    serialized_data: Vec::new(),
+                                    signature: operation.signature.to_string(),
+                                    content_creator_pub_key: operation.content_creator_pub_key.to_string(),
+                                    content_creator_address: operation.content_creator_address.to_string(),
+                                    id: operation.id.to_string()
+                                };
+                                // Send the new operation through the channel
+                                if let Err(e) = tx.send(Ok(grpc::NewOperationsStreamResponse {
+                                    id: request_id.clone(),
+                                    operation: Some(ret)
+                                })).await {
+                                    error!("failed to send operation : {}", e);
+                                    break;
+                                }
+                            },
+                            Err(e) => error!("{}", e)
+                        }
+                    },
+                    // Receive a new message from the in_stream
+                    res = in_stream.next() => {
+                        match res {
+                            Some(res) => {
+                                match res {
+                                    Ok(data) => {
+                                        // Update current filter && request id
+                                        filter = data.query
+                                        .and_then(|q| q.filter);
+                                        request_id = data.id;
+                                    },
+                                    Err(e) => {
+                                        error!("{}", e);
+                                        break;
+                                    }
+                                }
+                            },
+                            None => {
+                                // Client disconnected
+                                break;
+                            },
+                        }
+                    }
+                }
+            }
+        } else {
+            error!("empty request");
+        }
+    });
+
+    let out_stream = tokio_stream::wrappers::ReceiverStream::new(rx);
+    Ok(Box::pin(out_stream) as NewOperationsStream)
+}
+
+/// Return if the type of operation is filtered
```

Comment:
Is the term "filtered" the same throughout the codebase? It could mean "is to be shown" or "is to be hidden" depending on context I think. For instance is_filtered here returns `!filtered_ope_ids.contains(&id)` which does not seem consistant

Reply:
I'm agree that I find this little bit confusing too... cc @modship